lib/gax.js

```diff
@@ -55,6 +57,13 @@ var OPTION_INHERIT = exports.OPTION_INHERIT = {};
  * @param {PageDescriptor} settings.pageDescriptor - indicates the structure
  *   of page streaming to be performed. If set to null, page streaming
  *   is disabled.
+ * @param {boolean} settings.flattenPages - If there is no ``pageDescriptor``,
```

Comment:
well -- no.

there is {@link} tag in JSDoc, but it doesn't link to other parameters. So used {@link} for FIRST_PAGE, and this is replaced by `pageDescriptor`, which will be marked up as code section (when we introduce markdown plugin http://usejsdoc.org/plugins-markdown.html).